examples/persistent-storage/esp32/sdkconfig.defaults

```diff
@@ -25,3 +25,6 @@ CONFIG_ESPTOOLPY_MONITOR_BAUD=115200
 # Use a custom partition table
 CONFIG_PARTITION_TABLE_CUSTOM=y
 CONFIG_PARTITION_TABLE_FILENAME="partitions.csv"
+
+#Enable BT
+CONFIG_BT_ENABLED=y
```

Comment:
Why is BT enabled in `persistent-storage` example?

Reply:
Fixed.